docusaurus/docs/React/message-components/message-ui.mdx

```diff
@@ -113,6 +113,14 @@ while editing (overrides the value stored in `MessageContext`).
 | ------ |
 | object |
 
+### autoscrollToBottom
+
+Call this function to keep message list scrolled to the bottom when the message list container scroll height increases (only available in the `VirtualizedMessageList`). An example use case is that upon user's interaction with the application, a new element appears below the last message. In order to keep the newly rendered content visible, the `autoscrollToBottom` function can be called. The container, however, is not scrolled to the bottom, if already scrolled up more than 4px from the bottom.
```

Comment:
It's great if you can add a code example where the image onload event is wired up to scroll to the bottom.

Reply:
Thank you @petyosi I have added the example to the documentation